### ISSUE TITLE
bug: missing links between ontologies in case the base model has been added after sub-model

### ISSUE THREAD

**Issue by sszuev** [avicomp/ont-api#17]
this is possible if org.semanticweb.owlapi.model.MissingImportHandlingStrategy#SILENT is used

**Comment by sszuev** [comment]
also, NPE in case OntologyManager#addOntology is used:

```
java.lang.NullPointerException
	at java.util.Objects.requireNonNull(Objects.java:203)
	at java.util.Optional.<init>(Optional.java:96)
	at java.util.Optional.of(Optional.java:108)
	at java.util.stream.FindOps$FindSink$OfRef.get(FindOps.java:193)
	at java.util.stream.FindOps$FindSink$OfRef.get(FindOps.java:190)
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
	at ru.avicomp.ontapi.OntologyFactoryImpl$ONTLoaderImpl.fetchGraph(OntologyFactoryImpl.java:463)
	at ru.avicomp.ontapi.OntologyFactoryImpl$ONTLoaderImpl.makeUnionGraph(OntologyFactoryImpl.java:404)
	at ru.avicomp.ontapi.OntologyFactoryImpl$ONTLoaderImpl.createModel(OntologyFactoryImpl.java:331)
	at ru.avicomp.ontapi.OntologyFactoryImpl$ONTLoaderImpl.load(OntologyFactoryImpl.java:290)
	at ru.avicomp.ontapi.OntologyFactoryImpl.loadOWLOntology(OntologyFactoryImpl.java:108)
	at ru.avicomp.ontapi.OntologyFactoryImpl.loadOWLOntology(OntologyFactoryImpl.java:57)
	at ru.avicomp.ontapi.OntologyManagerImpl.load(OntologyManagerImpl.java:1656)
	at ru.avicomp.ontapi.OntologyManagerImpl.load(OntologyManagerImpl.java:1622)
	at ru.avicomp.ontapi.OntologyManagerImpl.loadOntologyFromOntologyDocument(OntologyManagerImpl.java:1565)
	at ru.avicomp.ontapi.OntologyManagerImpl.addOntology(OntologyManagerImpl.java:694)
```

**Comment by sszuev** [comment]
fixed (1.2.1)
